### DISCUSSION TITLE
Should different renovate runs (e.g. once for gitlab, once for bitbucket) use different baseDirs? Are they influencing each other?

### DISCUSSION THREAD

**Discussion by herndlm** [renovatebot/renovate#29103] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub, GitLab, Bitbucket

### Please tell us more about your question or problem

We have a set of shared runners and I have multiple renovate jobs (one for gitlab, one for github and one for bitbucket) and I was just wondering if I should set different https://docs.renovatebot.com/self-hosted-configuration/#basedir or https://docs.renovatebot.com/self-hosted-configuration/#cachedir to avoid the runs influencing each other.

Or what do you think is the way to run them and make use of the (shared) cache most efficiently?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
The {{baseDir}}/repos subfolder will put the platform first (e.g. "bitbucket/") so that should be OK to share.

I think cache is OK to share as long as only one of them runs at a time. Some cache items are not intended to support simultaneous write.

If in doubt, keep them completely separate.

**Reply by herndlm** [reply]
that makes sense, yeah. ok great, that was exactly what I wanted to know. I'm avoiding the parallel runtime anyway. thx!
